spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe 'UsersController', type: :request do
+  describe 'GET /users' do
+    before(:each) { get '/users' }
+
+    it 'should have status :success' do
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'should render :index' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'should return the user with id' do
+      expect(response.body).to include('Show all users')
+    end
+  end
+
+  describe 'GET /users/:id' do
+    before(:each) { get '/users/1' }
+
+    it 'should have status :success' do
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'should render :show' do
+      expect(response).to render_template(:show)
+    end
```

Comment:
- [ ] I am getting 4 errors, can you please double check them :pray:

Reply:
- [ ]  users_spec.rb and posts_spec.rb needs to be on the spec/requests/ folder, kindly update/create that folder. This way is better/easier to check and manage your tests

Reply:
Thank you for the suggestions. I will make the changes. However, regarding the tests that are not passing, I am not getting any errors on my end and I don't know how to visualize the errors. Perhaps, if I could get more context on this issue, I could be properly oriented toward ways I could fix it. Screenshot of my tests passing: 
![image](https://user-images.githubusercontent.com/107795045/216178950-a2fe2455-cbe1-42fe-a6f6-ce6faed2c761.png)